src/main/java/com/itechart/javalab/library/dao/impl/SqlBookDao.java

```diff
@@ -23,4 +37,50 @@ public static BookDao getInstance() {
         return instance;
     }
 
+
+    @Override
+    public Optional<List<Book>> getBooks() {
+
+        List<Book> books;
+
+        try (Connection connection = connectionPool.getConnection()) {
+            try (PreparedStatement preparedStatement = connection.prepareStatement(GET_ALL_BOOKS)) {
```

Comment:
can be written like this:  `try (Connection connection = connectionPool.getConnection(); PreparedStatement preparedStatement = connection.prepareStatement(GET_ALL_BOOKS)) {}`